test/mysql.test.js

```diff
@@ -88,7 +89,7 @@ describe('mysql', function() {
 
         p.content.should.be.equal(post.content);
         p.title.should.be.equal('a');
-        p.comments.should.eql(['1', '2']);
+        p.comments.should.eql(new List(['1', '2']));
```

Comment:
This change seems unrelated to the introduction of shared test suite from juggler v4. Is it needed to fix failing tests? Can we make it in a standalone PR please?